Mage.Sets/src/mage/cards/m/MythRealized.java

```diff
@@ -52,11 +56,11 @@ public MythRealized(UUID ownerId, CardSetInfo setInfo) {
 
         // {W}: Until end of turn, Myth Realized becomes a Monk Avatar creature in addition to its other types and gains "This creature's power and toughness are each equal to the number of lore counters on it."
         Effect effect = new BecomesCreatureSourceEffect(new MythRealizedToken(), null, Duration.EndOfTurn);
-        effect.setText("Until end of turn, {this} becomes a Monk Avatar creature in addition to its other types");
+        effect.setText("Until end of turn, {this} becomes a Monk Avatar creature in addition to its other types ");
         Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, effect, new ManaCostsImpl<>("{W}"));
-        ability.addEffect(new MythRealizedSetPTEffect(Duration.EndOfTurn));
-        this.addAbility(ability);        
-        
+        ability.addEffect(new SetPowerToughnessSourceEffect(loreCounterCount, Duration.EndOfTurn).setText("and gains \"This creature's power and toughness are each equal to the number of lore counters on it.\""));
```

Comment:
This effect constructor needs the `SubLayer.SetPT_7b` argument, because the ability is not a characteristic-defining ability.